cpp/memilio/io/README.md

```diff
@@ -140,14 +140,48 @@ To set the value of a parameter from the command line, first type the correspond
 
 ## The history object
 
-The `history.h` file defines a `History` class that handles writers and loggers for saving current parameters of our models.
+The history object provides a way to save data throughout a simulation. It provides an interface where one can specify which data of a given object should be saved through loggers and how it should be saved through writers.
```

Comment:
Good introduction!
The capitalization of Logger and Writer is inconsistent throughout this paragraph. Maybe use capital "Logger" when referring to some structs, and use e.g. "logging" instead of "logger use case" below? I am not firm on this, so if you have any idea please add it.

Also, technically the History is independent of the simulation - but here brevity is probably better than thinking of all possible use cases ;)